### ISSUE TITLE
 Could you provide the pre-trained files? I would be very grateful.

### ISSUE THREAD

**Issue by zly1213811** [zcablii/LSKNet#55]
### Model/Dataset/Scheduler description

This will be very helpful for my studies.

### Open source status

- [X] The model implementation is available
- [X] The model weights are available.

**Comment by zcablii** [comment]
Pretrained models are provided in [README.md](https://github.com/zcablii/LSKNet?tab=readme-ov-file#results-and-models).

In case you didn't notice, I copy and paste here:

Imagenet 300-epoch pre-trained LSKNet-T backbone: [model](https://liveuclac-my.sharepoint.com/:u:/g/personal/zcablii_ucl_ac_uk/EQ_R312q8ihJg1CSwhm80kgBNrxi1CRyLN8vXjF_lAszcA?e=fftQcx)

Imagenet 300-epoch pre-trained LSKNet-S backbone: [model](https://liveuclac-my.sharepoint.com/:u:/g/personal/zcablii_ucl_ac_uk/EYQMJzI0z0JEtCc-ebVWHbgBftwi0IltLMrbEoQT0qiSzw?e=hm1xKs)

**Comment by zly1213811** [comment]
翻译成英文是：

My meaning is the project files used for pre-training on the ImageNet dataset.

**Comment by zly1213811** [comment]
My meaning is the project files used for pre-training on the ImageNet dataset.

**Comment by zcablii** [comment]
I see. please refer to #7

**Comment by zly1213811** [comment]
Thank you very much!
